debezium-core/src/test/java/io/debezium/junit/ConditionalFail.java

```diff
@@ -55,4 +72,37 @@ else if (failure != null) {
             throw new IllegalStateException(e);
         }
     }
+
+    private Statement ignoreFlakyFailure(final Statement base, final Description description,
+                                         final Flaky flakyClass) {
+
+        final String flakyFailuresProperty = System.getProperty(Flaky.IGNORE_FLAKY_FAILURES_PROPERTY);
+        if (flakyFailuresProperty == null || !Boolean.valueOf(flakyFailuresProperty)) {
+            return base;
+        }
+        final String flakyAttemptsProperty = System.getProperty(Flaky.FLAKY_ATTEMPTS_FAILURES_PROPERTY, "1");
```

Comment:
I like this idea, and it raises a question for me.

Should we consider adding a workflow that we can manually trigger where we start a CI build for the specific connector module and be able to specify the test class and/or method name to re-run the test on the GitHub CI infrastructure?  If so, adding this to that workflow might be neat so we have a better way to combat testing on GitHub CI with race conditions rather than waiting 30-60 minutes for an entire test suite build.  What do others think?

Reply:
Yes, such workflow would be definitely useful.

Reply:
Good proposal, logged via https://issues.redhat.com/browse/DBZ-6333